scene/3d/skeleton_3d.cpp

```diff
@@ -318,6 +318,9 @@ void Skeleton3D::_notification(int p_what) {
 				set_physics_process_internal(true);
 			}
 		} break;
+		case NOTIFICATION_NODE_RECACHE_REQUESTED: {
```

Comment:
This should be removed now. And the notification itself can be removed completely (don't forget to run the doctool).

Reply:
Ah, missed this. Removed it now.